### DISCUSSION TITLE
radio presets dont save on round change

### DISCUSSION THREAD

**Discussion by nagaisa** [FakeFishGames/Barotrauma#14168] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Using them mem button to set a radio preset, resets the preset back to default when round changes.
![before round change](https://github.com/FakeFishGames/Barotrauma/assets/82787802/945d7699-32ac-4f7a-a2c4-3e5455673dd7)
![After round change](https://github.com/FakeFishGames/Barotrauma/assets/82787802/2180ff76-44b3-4554-9523-e6b3a987b9c5)

### Reproduction steps

1. start a campaign save 2. change radio fq to something 3. save that fq useing the mem and a preset 4.switch rounds

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now. Fix coming up in the next unstable build and the upcoming release.
